Snowflake.Data.Tests/IntegrationTests/SFBindTestIT.cs

```diff
@@ -1,6 +1,7 @@
 ﻿/*
  * Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
  */
+#nullable enable
```

Comment:
shouldn't we do `#nullable disable` at the end of the file?

Reply:
added `restore`